### DISCUSSION TITLE
React js integration

### DISCUSSION THREAD

**Discussion by mayankpandey-gt** [lowdefy/lowdefy#1129] [General]
How we can integrate this app with reactjs without using new port.

**Comment by Gervwyk** [comment]
Hi @mayankpandey-gt 

It's a really cool idea to pacakge a lowdefy app as a react component for further integration, currently not possible, unfortunately. However will be a really nice use case to cover, to do it we'll need to develop a react component that renders a "page" or element instead of building the entire front end.  For v4, we are building lowdefy as a next.js app, for this use case, it will be cool to have the next.js backend, but instead of building the next.js front end, we can export a Lowdefy component that mounts the application in a react component. 

It will still require that you need to send requests for the backend to server to component content, however, when we look at static deployments, we can consider a build command which exports the component with the config included.
